src/main/navigation/navigation.c

```diff
@@ -2693,6 +2693,11 @@ rthState_e getStateOfForcedRTH(void)
     }
 }
 
+bool navigationIsControllingThrottle(void)
+{
+    return posControl.navState != NAV_STATE_IDLE;
```

Comment:
This is not always true. For multirotors we have 2D poshold mode which does not control throttle.

It's better to check for result of a call to `navGetCurrentStateFlags()` and also mind fixed-wing:
```
navigationFSMStateFlags_t stateFlags = navGetCurrentStateFlags();
return (stateFlags & (NAV_CTL_ALT | NAV_CTL_EMERG | NAV_CTL_LAUNCH | NAV_CTL_LAND)) || (STATE(FIXED_WING) && (stateFlags & (NAV_CTL_POS)));
```